README.md

```diff
@@ -354,6 +354,7 @@ Developer note: The software implements the [Torznab](https://github.com/Sonarr/
  * MoeCat
  * Mononoké-BT
  * MoreThanTV (MTV)
+ * MMA-Torrents
```

Comment:
before Mononoké-BT

Reply:
Alphabetically, it should be between MicroBit and MoeCat looks like

Reply:
right, I'm going to sleep, this is a clear signal...